pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1426,8 +1426,11 @@ func buildHTTPConnectionManager(node *model.Proxy, env *model.Environment, httpO
 			Name: xdsutil.FileAccessLog,
 		}
 
-		if util.IsXDSMarshalingToAnyEnabled(node) {
+		if util.IsProxyVersionGE11(node) {
 			buildAccessLog(fl, env)
+		}
+
+		if util.IsXDSMarshalingToAnyEnabled(node) {
 			acc.ConfigType = &accesslog.AccessLog_TypedConfig{TypedConfig: util.MessageToAny(fl)}
 		} else {
```

Comment:
else if IsProxyVersionGE11

Reply:
`buildAccessLog` generates the 1.1 access log configuration. So we need to run that only if version is 1.1. We need to set the overall access log config no matter what version.